src/components/LoginForm.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components/macro";
+import icon from "../assets/icons/login_black_24dp.svg";
+import Button from "./Button";
+
+export default function LoginForm() {
+
+	function login(event) {
+		event.preventDefault();
+		const form = event.target;
+		form.reset()
+	}
+
+	return (
+		<StyledForm onSubmit={(event) => login(event)}>
+			<StyledLabel>
+				Benutzername
+				<StyledInput
+					type="text" 
+					id="login__name" 
+					name="login__name" 
```

Comment:
Please keep the name in camelCase as we are using it later on and it's quite un-commen to have a name like this. Also, just `name` would be enough.
```suggestion
					name="name" 
```